brain-marks/Localization/id.lproj/Localizable.strings

```diff
@@ -0,0 +1,55 @@
+/* 
+  Localizable.strings
+  brain-marks
+
+  Created by AC Richter on 13.12.21. 
```

Comment:
@alfinsyahruddin I see you copied one of the first ones, which is totally fine!
But you could change your name in the comment if you want! Plus technically the comment would be incorrect right now, cause it wasn't created by this person

Reply back here and I can change it before merging!